app/javascript/react/types/dateFormat.ts

```diff
@@ -0,0 +1,6 @@
+enum DateFormat {
+    default = "YYYY-MM-DD",
+    us = "MM/DD/YYYY",
```

Comment:
Just thinking out loud - why do we have two differents formats, not just the US one? Asking because I saw Michael's comment that he wants to have an American date format 🤔

Reply:
Good point, so I just checked to ensure myself. The reason is that the backend sends us the date in default format (let's name it like that). To receive it we need to have default type, and US type is needed - you know why 😄